### ISSUE TITLE
_an-object.js:3 - Uncaught TypeError: r is not a function after build

### ISSUE THREAD

**Issue by ahmadina** [mxstbr/react-boilerplate#1395]
Hi, after run command build (npm run build) and open project in browser (chrome, firefox, ...) I saw this error:

~~~
Uncaught TypeError: r is not a function
    at e.exports (_an-object.js:3)
    at e.exports (_an-object.js:3)
    at Object../node_modules/core-js/modules/es6.object.to-string.js (es6.object.to-string.js:5)
    at t (bootstrap 703288e…:68)
    at Object../node_modules/core-js/shim.js (shim.js:18)
    at t (bootstrap 703288e…:68)
    at Object.<anonymous> (index.js:3)
    at Object../node_modules/babel-polyfill/lib/index.js (main.2b538db….js:3101)
    at t (bootstrap 703288e…:68)
    at Object../app/app.js (main.2b538db….js:1540)
~~~

everybody can help me what's can I do?

**Comment by gihrig** [comment]
Tell us more about your environment, version of boilerplate, node, npm, OS, 

What mods have you made?

What are the full commands you used to get to the error?

Did it ever work for you? What changed that caused the error, etc.

**Comment by ahmadina** [comment]
- Babel version: 6.1.18
- Node version: 6.4.0
- npm version: 3.10.3

@gihrig This error did earlier. Recently come, in production mode after run command "npm run build"
I'm just append react-bootstrap all things in project are default

**Comment by gihrig** [comment]
I'm not getting that error when I 
- clone new project
- change to dev branch
- run `yarn`
- run `npm run build`
- run `npm run start:prod`
- open `localhost:3000` in Chrome

So, I'm guessing the problem is related to installing `react-bootstrap`. Have seen this [discussion](https://github.com/styled-components/styled-components/issues/293) on the styled-components project?

**Comment by kelsonic** [comment]
I'm getting the same issue when using `react-bootstrap` with your boilerplate. 

In `./app/app.js` I added:
```js
// Import Bootstrap styling
import 'bootstrap/dist/css/bootstrap.css';
```
I also added `"react-bootstrap": "^0.30.7",` to the `package.json` and a couple bootstrap components inside of containers (Happy to share the specifics if it would be helpful).

When I run `npm i && npm start` it runs just fine on my local. However, when I do:

> - run yarn
> - run npm run build
> - run npm run start:prod
> - open localhost:3000 in Chrome

it throws this error:

```
main.f600936….js:1 Uncaught TypeError: getOwnPropertyDescriptor is not an object!
    at e.exports (main.f600936….js:1)
    at Object../node_modules/core-js/modules/es6.object.get-own-property-descriptor.js (main.f600936….js:9)
    at t (main.f600936….js:1)
    at Object../node_modules/core-js/shim.js (main.f600936….js:11)
    at t (main.f600936….js:1)
    at Object.<anonymous> (main.f600936….js:2)
    at Object../node_modules/babel-polyfill/lib/index.js (main.f600936….js:2)
    at t (main.f600936….js:1)
    at Object../app/app.js (main.f600936….js:1)
    at t (main.f600936….js:1)
```

Any help would be greatly appreciated! :)

**Comment by kelsonic** [comment]
I figured out the issue. After deleting [this line](https://github.com/mxstbr/react-boilerplate/blob/master/internals/webpack/webpack.prod.babel.js#L28) in `webpack.prod.babel.js` it works.

**Comment by gihrig** [comment]
@kelsonic 

>After deleting this line in webpack.prod.babel.js it works.
new webpack.optimize.DedupePlugin(),

Good sleuthing! 😄 

Dedup-plugin was removed in #1218. This change will be included in the upcoming 3.4 release.
